getting-started.md

```diff
@@ -31,11 +34,11 @@ The {{site.data.keyword.speechtotextfull}} service transcribes audio to text to
 
 -   {: download} If you're seeing this text, you created your service instance. Now get your credentials.
 -   Create an instance of the service:
-    1.  Go to the [{{site.data.keyword.speechtotextshort}} ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://console.{DomainName}/catalog/services/speech-to-text){: new_window} page in the {{site.data.keyword.Bluemix_notm}} Catalog.
+    1.  Go to the [{{site.data.keyword.speechtotextshort}} ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://{DomainName}/catalog/services/speech-to-text){: new_window} page in the {{site.data.keyword.Bluemix_notm}} Catalog.
```

Comment:
@jeffpk62 Please check after building to make sure the url works to validate my own testing.

Reply:
@SirSpidey OK, I'll take a look when it's update; thanks.